lumicks/pylake/force_calibration/calibration_models.py

```diff
@@ -158,13 +159,16 @@ class PassiveCalibrationModel:
         Enable hydrodynamic correction.
     distance_to_surface : float, optional
         Distance from bead center to the surface [um].
-        Currently not supported for models without hydrodynamics.
+        When specifying `None`, the model will use an approximation which is only suitable for
+        measurements performed deep in bulk.
```

Comment:
Now that this is supported without hydro, the check for `hydrodynamically_correct` on line 192 has to be removed.

Reply:
Damn! Good catch!